test/lux/basic.lux

```diff
@@ -183,6 +183,24 @@
     ?KeyboardInterrupt
     ~$_CTRL_C_
     ?SH-PROMPT:
+[shell ncs-cli]
+    [timeout 60]
```

Comment:
You should not need such a long timeout, and definitely not before the `explore-transitions` actions. Moreover, you forgot to clean the timeout.

Reply:
changed to 10 and cleaned at the end